HM2task1.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#define SIZE 47
+
+int recursivelyFibonacci(int num)
+{
+    if (num <= 1)
+    {
+        return 1;
+    }
+    return recursivelyFibonacci(num - 2) + recursivelyFibonacci(num - 1);
+}
+
+int iteractiveFibonacci(int num)
+{
+    int currentNum = 1;
+    int previousNum = 1;
+    for (int i = 2; i <= num; i++)
+    {
+        int temp = currentNum;
+        currentNum = previousNum + currentNum;
+        previousNum = temp;
+    }
+    return currentNum;
+}
+
+int main()
+{
+    int number = 0;
+    int scan_res = 0;
+    bool isIncorrectInput = true;
+
+    do
+    {
+        isIncorrectInput = true;

```

Comment:
```suggestion
        isСorrectInput = false;
```
В коде принято мыслить позитивно. Ибо получается, что в переменную отрицание вшито